### PR TITLE
feat: On air segment history

### PR DESCRIPTION
**Pull request by thomasslee97**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR introduces a `followOnAirSegmentsHistory` setting, which modifies the behaviour of scrolling to a segment to allow a history of segments to be shown. e.g. if `followOnAirSegmentsHistory` is 0, you will scroll to the segment - the current behaviour. If `followOnAirSegmentsHistory` is set to 1, the scroll will scroll to show the selected segment and the segment before it - showing one segment of history. This is a feature requested by TV 2 so that their users can see segments within the context of what was played before.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

**Status**

- [ ] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [X] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/client/lib/viewPort.ts [hunk 1]
```diff
@@ -113,14 +114,60 @@ export function scrollToSegment(
...
+	const getElementToScrollTo = (showHistory: boolean): HTMLElement | null => {
+		if (isProtectedString(elementToScrollToOrSegmentId)) {
+			console.log(elementToScrollToOrSegmentId)
+			let targetElement = document.querySelector<HTMLElement>(
+				`#${SEGMENT_TIMELINE_ELEMENT_ID}${elementToScrollToOrSegmentId}`
+			)
+
+			console.log(targetElement)
+			console.log(Settings.followOnAirSegmentsHistory)
+			console.log(targetElement?.parentElement?.parentElement?.previousElementSibling)
```
Comment (jstarpl) [inline meteor/client/lib/viewPort.ts:126]: I suppose these should be removed?

**Review by jstarpl — CHANGES_REQUESTED** [review]
Great new feature! Just a single comment regarding the leftover `console.log`'s

File: meteor/client/lib/viewPort.ts (hunk 1 above)
Comment (thomasslee97) [reply meteor/client/lib/viewPort.ts]: Ah! Yes, I have removed them here https://github.com/olzzon/tv-automation-server-core/commit/16eff9414cc2d34d93707c8fc0d6eebbb6ca57cd

**Comment by thomasslee97** [comment]
The unused property removed here https://github.com/nrkno/tv-automation-server-core/pull/401/commits/f48fef8517e81eb856e0eb781918477b58e9cc09 has been moved to the inner function

**Review by jstarpl — APPROVED** [review]
Just pending acceptance now.

**Pull request merged**